lib/jwt/jwt-authenticator.js

```diff
@@ -29,7 +29,7 @@ JwtAuthenticator.prototype.withCookie = function withCookie(cookieName){
 };
 
 JwtAuthenticator.prototype.unauthenticated = function unauthenticated(){
-  return new ApiAuthRequestError('Unauthorized', 401);
+  return new ApiAuthRequestError({userMessage:'Unauthorized', statusCode: 401});
```

Comment:
I agree that we could refactor these error responses to be less dry.  Our eventual goal is to deprecate `application.authenticateApiRequest()` in favor of our new authenticator pattern, i.e. `new Authenticator(application)`, etc.  We can revisit the error strategy when that happens.